libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -52,6 +52,8 @@ void NavEKF2_core::readRangeFinder(void)
                 }
                 storedRngMeasTime_ms[sensorIndex][rngMeasIndex[sensorIndex]] = imuSampleTime_ms - 25;
                 storedRngMeas[sensorIndex][rngMeasIndex[sensorIndex]] = sensor->distance_cm() * 0.01f;
+            } else {
+                continue;
```

Comment:
not a good one for backport

Reply:
This was already approved above, and fixes a safety bug